app/components/UI/Button.js

```diff
@@ -60,13 +60,16 @@ class Button extends React.PureComponent {
 
     const borderRadius = variant === 'secondary' ? 0 : 5
 
+    //support custom styled and styled-components
+    const wrapperClasses = [className, active ? 'active' : null].filter(cls => !!cls).join(' ')
```

Comment:
Prefer `Boolean()` rather than `!!` for better readability.

Reply:
Fixed